### PR TITLE
Ds ethiop12new

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some revisions.

### CODE REVIEW

File: DayrSuryan/DSEthiop12.xml [hunk 1]
```diff
@@ -559,18 +562,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                           It has varying appearance in different parts of the manuscript
-                            up to an impression that more than one hand
-                          participated in writing. But it was most probably only one scribe (with
-                          possible exception for <locus from="80v" to="81r"/>),
-                          using the same script (broadly spaced, nearly upright,
-                         with broad rectangular letter shapes).
-                          The parts of the manuscript were possibly copied at different times.
-                          Hence differences in the hues of the inks, slant and thickness of the strokes,
-                          quality of the execution (diminishing after ca. <locus target="#45"/>).
+                        <date notBefore="1800" notAfter="1899">20th century</date>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:565]: 19th century then

File: DayrSuryan/DSEthiop12.xml [hunk 2]
```diff
@@ -580,12 +580,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <seg type="ink">Black, violet.</seg>
                         <seg type="rubrication">Beginning and end.</seg>
-                        <date notBefore="1900" notAfter="1954">20th century</date>
-                        <desc>The handwriting of only the supplication on <locus from="80v" to="81r"/>,
-                        more careful as compared to <ref target="#h1"/>. The script is
-                        slender, right-slanted, broadly spaced, with tendency to rounded shapes.
-                        Despite the differences, it may still be the same <ref target="#h1"/>, writing in different conditions and
-                        with different instruments.
+                        <date notBefore="1800" notAfter="1899">19th century</date>
+                        <desc>BEtter trained but not very regular hasty hand.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:584]: Better

File: DayrSuryan/DSEthiop12.xml [hunk 3]
```diff
@@ -732,13 +732,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                before 1954</origDate>.
-                The parts of the manuscript could be written by the same
-                scribe in Egypt, but at different times, trimmed and bound together
-
-                 sometime around <date>1950</date>, but before <date>1954</date>).
-                  The owner and possibly the scribe was <persName ref="PRS11905GabraM">Gabra Māryām of Gondar</persName>.
+              <origDate notBefore="1840" notAfter="1870" evidence="lettering">Ca. 1840-1870</origDate>.
+              Parts of the manuscript wer written in Egypt, over a period of time,
+               later they were gathered, trimmed and bound together.
+                 The main <ref target="#h1"/> seems to be identical to
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:738]: I think it is better to write in the content of the refs what we want to have printed, so The `<ref target="#h1">main hand</ref>` or `<ref target="#h1">Hand 1</ref>` , etc. also for the following refs in this paragraph

**Comment by thea-m** [comment]
Interesting!

File: DayrSuryan/DSEthiop12.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:565]: Ok.

File: DayrSuryan/DSEthiop12.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:584]: Ok

File: DayrSuryan/DSEthiop12.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:738]: Like this (see new version)?

Comment (thea-m) [reply DayrSuryan/DSEthiop12.xml]: perfect, I think

**Pull request merged**